packages/next/client/head-manager.js

```diff
@@ -46,9 +46,17 @@ export default class HeadManager {
 
   updateElements (type, components) {
     const headEl = document.getElementsByTagName('head')[0]
-    const oldTags = Array.prototype.slice.call(
-      headEl.querySelectorAll(type + '.next-head')
-    )
+    const headCountEl = headEl.querySelector('meta[name=next-head-count]')
+    const headCount = Number(headCountEl.content)
+    const oldTags = []
+
+    let currentEl = headCountEl.previousElementSibling
+    for (let i = 0; i < headCount; i++) {
```

Comment:
Looks like we can save a few bytes if this is renamed to `j`.

Reply:
Sorry, GitHub doesn't make it easy to see exactly what you're referring to. Do you mean `i`, `headCount`, or `currentEl` should be `j`? In the case of `headCount`, it's being used later.

Reply:
I specifically meant (prev `j`, but it's now used):
```suggestion
    for (let k = 0; k < headCount; k++) {
```